catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventHandlerDelegate.java

```diff
@@ -348,7 +359,8 @@ public void error(SAXParseException exception) throws SAXException {
     @Override
     public void fatalError(SAXParseException exception) throws SAXException {
         String message = "Fatal Error: " + getParseExceptionInfo(exception);
```

Comment:
Should message be broken up and outWriter.appended as well?

Also just curious if you manually made these changes or if it was some formatter suggestions.